pkg/k8s/watchers/cilium_endpoint_batch.go

```diff
@@ -0,0 +1,84 @@
+//  Copyright 2021 Authors of Cilium
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package watchers
+
+import (
+	"sync"
+
+	"github.com/cilium/cilium/pkg/k8s"
+	cilium_v2a1 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
+	"github.com/cilium/cilium/pkg/k8s/informer"
+	"github.com/cilium/cilium/pkg/k8s/watchers/subscriber"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	cebNotify = subscriber.NewCEB()
+)
+
+func (k *K8sWatcher) ciliumEndpointBatchInit(client *k8s.K8sCiliumClient, asyncControllers *sync.WaitGroup) {
+	log.Info("Initializing CEB controller")
+	var once sync.Once
+
+	// Register for all ceb updates.
+	cebNotify.Register(newCEBSubscriber(k))
+
+	_, cebController := informer.NewInformer(
```

Comment:
I think it's best to rename this to make it clearer that it's not the controller itself.
```suggestion
	_, cebInformer := informer.NewInformer(
```

Reply:
Done.